src/worker.py

```
@@ -1301,4 +1337,145 @@ async def on_fetch(request, env):
+    body, bad = await parse_json_object(req)
+    if bad:
+        return bad
+    raw_msg = body.get("message")
+    message = (raw_msg.strip()[:200] if isinstance(raw_msg, str) else "")
+    conn_id = new_id()
+    try:
+        await env.DB.prepare(
+            "INSERT INTO peer_connections (id, requester_id, addressee_id, message)"
+            " VALUES (?, ?, ?, ?)"
+        ).bind(conn_id, user["id"], addressee_id, message).run()
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)